pattoo/db/schema/auth.py

```diff
@@ -0,0 +1,71 @@
+"""pattoo ORM Schema for User Authentication"""
+
+# PIP3 imports
+import graphene
+from graphql import GraphQLError
+from flask_graphql_auth import (create_access_token, create_refresh_token,
+                                get_jwt_identity,
+                                mutation_jwt_refresh_token_required)
+
+# pattoo imports
+from pattoo.db.table.user import User as UserTable
+
+
+class AuthMutationInput(graphene.InputObjectType):
+    """Arguments for user authentication"""
```

Comment:
Add period to docstring. This is important as we will eventually need it for the automatic creation of documentation.